docker-wrappers/OmicsIntegrator2/Dockerfile

```diff
@@ -2,5 +2,14 @@
 # https://github.com/fraenkel-lab/OmicsIntegrator2
 FROM continuumio/miniconda3:4.9.2
 
+COPY 0001-disable-graph-exports.patch .
+
+RUN git clone https://github.com/agitter/OmicsIntegrator2 && \
```

Comment:
Applying a patch can be a useful pattern, so a comment explaining what is happening here will help future maintainers.

Alternatively, we could use the new docs in #367 to write a short section on how to create and apply a patch, which would be even better.